### PR TITLE
story:30 bugs

### PR DESCRIPTION
**Pull request by IgorKhlebnikoff**
screenshot from 2015-03-09 16 21 16
screenshot from 2015-03-09 16 21 35
screenshot from 2015-03-09 16 22 53
screenshot from 2015-03-09 16 23 07
screenshot from 2015-03-09 16 23 15
screenshot from 2015-03-09 16 23 27

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -29,10 +29,11 @@ sendRequest = function(url, type) {
 Popup = function() {};
 
 Popup.hideModal = function() {
-  addWall = function(node) {
```
Comment (pavel-mihailovskiy) [inline app/assets/javascripts/application.js:32]: Why did you remove this??! it just should be added only when it loads

File: app/views/shared/_modal_window.html.slim [hunk 2]
```diff
@@ -1,3 +1,4 @@
 #modal_window.modal.fade
   .modal-dialog
     .modal-content
+#no_modal_window
```
Comment (pavel-mihailovskiy) [inline app/views/shared/_modal_window.html.slim]: where do you use it?

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (IgorKhlebnikoff) [inline app/assets/javascripts/application.js:32]: screenshot from 2015-03-09 17 36 29
screenshot from 2015-03-09 17 36 36

Comment (IgorKhlebnikoff) [inline app/assets/javascripts/application.js:32]: screenshot from 2015-03-09 17 41 12
